bin/dev.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+
+node "%~dp0\dev" %*
```

Comment:
```suggestion
node "%~dp0\dev" %*

```

Reply:
updated in https://github.com/nimblehq/infrastructure-templates/pull/22/commits/a89d823981614c070c2354964efc9293963a116d